linker/src/lib.rs

```diff
@@ -672,7 +679,7 @@ machine Main {
     pol constant p_read_X_pc = [0]*;
     pol constant p_reg_write_X_A = [0]*;
     { pc, reg_write_X_A, instr_add5_into_A, instr__jump_to_operation, instr__reset, instr__loop, instr_return, X_const, X_read_free, read_X_A, read_X_pc } in { p_line, p_reg_write_X_A, p_instr_add5_into_A, p_instr__jump_to_operation, p_instr__reset, p_instr__loop, p_instr_return, p_X_const, p_X_read_free, p_read_X_A, p_read_X_pc };
-    instr_add5_into_A { 0, X, A' } in main_vm.latch { main_vm.operation_id, main_vm.x, main_vm.y };
+    instr_add5_into_A * 1 { 0, X, A' } in main_vm.latch { main_vm.operation_id, main_vm.x, main_vm.y };
```

Comment:
Can these multiplications by 1 be avoided (at least in the Display routine)?

Reply:
I actually thought about modifying display, but this is best left to the optimizer I think: the other side of the multiplication by 1 could be anything.
Another option is having the link flag be an Option, but then it needlessly complicates the rest of code with option handling.
Maybe the "combine flags" function you suggested above would be a better place.

Reply:
done the above, worked nicely